chart/flux/README.md

```diff
@@ -190,7 +190,7 @@ The following tables lists the configurable parameters of the Weave Flux chart a
 | `git.label` | Label to keep track of sync progress, used to tag the Git branch | `flux-sync`
 | `git.ciSkip` | Append "[ci skip]" to commit messages so that CI will skip builds | `false`
 | `git.pollInterval` | Period at which to poll git repo for new commits | `5m`
-| `git.timeOut` | Duration after which git operations time out | `20s`
+| `git.timeout` | Duration after which git operations timeout | `20s`
```

Comment:
Now I feel like a pedant, but the verb is "to time out"

Reply:
~Never drop out of high school to pursue your dreams as an engineer kids, it will bite you later~

Changed like it never happened